widgets/linechart/internal/axes/value.go

```diff
@@ -49,12 +54,19 @@ func (v *Value) String() string {
 // NewValue returns a new instance representing the provided value, rounding
 // the value up to the specified number of non-zero decimal places.
 func NewValue(v float64, nonZeroDecimals int) *Value {
+	return NewFormattedValue(v, nonZeroDecimals, nil)
+}
+
+// NewFormattedValue returns a new instance representing the provided value,
+// using a value formatter.
+func NewFormattedValue(v float64, nonZeroDecimals int, formatter valueFormatter) *Value {
```

Comment:
(optional) Instead of creating a new constructor, we could add optional variadic arguments to NewValue which is what most of the Termdash repo does. Them being variadic means that none of the existing call sites would break.

Here is an example:
https://github.com/mum4k/termdash/blob/a9515f27215c8c7f3c0df731c4e86103cb9cdb9d/internal/draw/rectangle.go#L27-L65
What do you think?